core/src/main/java/nl/nn/adapterframework/util/TransformerPool.java

```diff
@@ -410,6 +411,10 @@ public String transform(Document d, Map<String,Object> parameters)	throws Transf
 		return transform(new DOMSource(d),parameters);
 	}
 
+	public String transform(Message m, Map<String,Object> parameters) throws TransformerException, IOException, SAXException {
```

Comment:
Daar is die dan eindelijk! Maar wat nu als je een SAXSource wilt hebben?

Kan String nu gedeprecate worden? (standaard staat resolveExternalEntities op false?)

Reply:
Het is meer dan ik liever niet wil dat we op meerdere plekken dingen bij moeten gaan houden.
Als hij niet gedeprecate wordt, dan op zijn minst hier new Message(input)
We moeten van de vele transforms en stringToSourceForSingleUse 'converters' af.

Reply:
Ga ik nu nog even niet aanpassen. Later wel...